lib/model/message.dart

```diff
@@ -60,14 +60,17 @@ class MessageStoreImpl with MessageStore {
     }
   }
 
-  void dispose() {
-    // When a MessageListView is disposed, it removes itself from the Set
-    // `MessageStoreImpl._messageListViews`. Instead of iterating on that Set,
-    // iterate on a copy, to avoid concurrent modifications.
-    for (final view in _messageListViews.toList()) {
-      view.dispose();
-    }
-  }
+  // No `dispose` method, because there's nothing for it to do.
```

Comment:
nit:

    2. Being dependent on `GlobalStore`, `PerAccountStoreWidget` is rebuilt.
       This time, the StatefulElement corresponding to the `MessageList`
       widget, is no longer in the element tree because `PerAccountStoreWidget`
       cannot find the account and builds a placeholder instead.

It'd be good to call out explicitly at the start of this scenario (before the numbered steps) that you're talking about a case where `removeAccount` is called, like if the user logs out. I read this step a couple of times and initially thought it was mistaken, because the main scenario that comes to my mind for a store getting disposed is when it's replaced by a new store.